Homework 9/Ex1/Program.cs

```diff
@@ -0,0 +1,59 @@
+﻿class DynamicArrayService
+{
+    private List<int> arrayList = new List<int>();
+
+    public void AddElement(int element)
+    {
+        arrayList.Add(element);
+        Console.WriteLine("Current number of elements: " + arrayList.Count);
+    }
+
+    public void RemoveElement(int value)
+    {
+        arrayList.Remove(value);
+        Console.WriteLine("Remaining elements: " + string.Join(", ", arrayList));
+    }
+
+    public void FillWithRandomNumbers()
+    {
+        Random random = new Random();
+        for (int i = 0; i < 10; i++)
+        {
+            arrayList.Add(random.Next(1, 101));
+        }
+    }
+
+    public void SearchElement(int value)
+    {
+        int index = arrayList.IndexOf(value);
+        if (index != -1)
+        {
+            Console.WriteLine("Element found at index: " + index);
+        }
+        else
+        {
+            Console.WriteLine("Element not found");
+        }
```

Comment:
Это можно сделать короче:
```suggestion
        Console.WriteLine(index != -1 ? $"Element found at index: {index}" : "Element not found");
```